source/mesh_refinement/particle_density.cc

```diff
@@ -54,7 +54,7 @@ namespace aspect
             // Note that measure() only gives a linear approximation of the cell's volume.
             // This is unaccurate for curved cells, but the exact measure is likely not
             // important for this plugin since the cell distortion is usually small.
-            indicators(i) = n_tracers / cell->measure();
+            indicators(i) = static_cast<float>(n_tracers);
```

Comment:
Hm, I don't think I quite understand what you're doing here. You just consider the particle number now, but no longer divide by the measure. How do you go from here to a *density* as the name of the plugin suggests? It's also no longer current with the comment above it.

By the way, typo: unaccurate -> inaccurate